src/main/java/com/elixr/poc/Constant/Constants.java

```diff
@@ -0,0 +1,9 @@
+package com.elixr.poc.Constant;
+
+/* this constant file holds all the error messages for all the end points */
+
+public class Constants {
```

Comment:
Rename this class as ApplicationConstants.java

Reply:
Updated